examples/tutorials/Training_an_Exchange_Correlation_Functional_using_Deepchem.ipynb

```diff
@@ -7,78 +7,49 @@
    "source": [
```

Comment:
<p>This content and references should not be removed. We should not lose content in making tutorials</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think the original text is too much jargon, if you could simplify a bit, that would be good, especially the third paragraph.</p>